src/Controls/src/Core/HandlerImpl/WebView/WebView.Windows.cs

```diff
@@ -1,9 +1,21 @@
-﻿using Microsoft.Maui.Controls.Platform;
-using Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific;
+﻿#nullable enable
+using System;
 
 namespace Microsoft.Maui.Controls
 {
 	public partial class WebView
 	{
+		protected override void OnPropertyChanging(string? propertyName = null)
```

Comment:
I think @PureWeen _just_ added a WindowChanged event - and we may want to make that public. Maybe when a window is closing, we can propagate a Window = null change?